ledger/participant-integration-api/src/test/suite/scala/platform/store/cache/MutableCacheBackedContractStoreSpec.scala

```diff
@@ -101,11 +101,15 @@ class MutableCacheBackedContractStoreSpec
         ).asFuture
         created1 <- createdEvent(cId_1, contract1, Some(someKey), Set(charlie), offset1, t1)
         _ <- eventually {
-          store.contractsCache.get(cId_1) shouldBe Some(Active(contract1, Set(charlie), t1))
-          store.keyCache.get(someKey) shouldBe Some(Assigned(cId_1, Set(charlie)))
+          store.contractStateCaches.contractState.get(cId_1) shouldBe Some(
```

Comment:
as you extracted out this functionality into a separate unit, shall we have a separate test for `ContractStateCaches` ? unless there is future work which is going to refactor it even further of course :)